src/storage/mvcc/reader.rs

```diff
@@ -273,6 +273,22 @@ impl<'a> MvccReader<'a> {
         Ok(())
     }
 
+    pub fn seek_first(&mut self, ts: u64) -> Result<Option<Key>> {
+        assert!(self.scan_mode.is_some());
+        self.create_write_cursor()?;
+
+        let mut w_cur = self.write_cursor.as_mut().unwrap();
+        let mut w_ok = w_cur.seek_to_first(&mut self.statistics.write);
+
+        while w_ok {
```

Comment:
I prefer to use `cursor.valid()` instead, then the variable `cursor` can be removed.